### DISCUSSION TITLE
How to prevent Google Auth from logging out

### DISCUSSION THREAD

**Discussion by mozeryansky** [supabase/supabase#18880] [Questions]
I'm using Supabase Auth with Google Sign In. Every few days it appears my token becomes invalid and I'm logged out. Is this expected or is there something else I need to do to make sure the token stays valid?

**Comment by GaryAustin1** [comment]
Are you talking about Supabase Auth staying signed in or using the Google token?
A signout from any device will be default signout all devices on next token refresh.
Normally Supabase itself stays signed in.  You may need to provide more info on your environment.

**Reply by mozeryansky** [reply]
I'm using `import { Auth } from '@supabase/auth-ui-react'` with a google provider. There's nothing else custom, just from the example repos. I don't intentionally signout, it's just sometimes I'll noticed my auth becomes invalid and I get supabase errors, and when I refresh my website I see I was logged out. It seems random.

What info about my environment can I provide?
